docs/Setup/setup-git-for-local-pr-review.md

```
@@ -0,0 +1,20 @@
+# Setup Git for local PR review
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)